gulpfile.js

```diff
@@ -77,6 +76,9 @@ gulp.task('styles', ['styles:fabricator', 'styles:toolkit']);
 
 
 // scripts
+var webpackCompiler = webpack(require('./webpack.config')(config));
```

Comment:
@LukeAskew I can move it, but it need to be below the config block. We could move the config to the very top?